index.html

```diff
@@ -146,6 +146,43 @@ <h3>Multi-Post Stories</h3>
         </ul>
         <button class="btn">See Project</button>
     </section>
+    <footer>
+        <h2 id="conttitle">CONTACT US</h2>
+        <p id="conttext">If you have an application you are interested in 
+            developing, a feature that you need built or a project that needs coding.
+             I’d love to help with it </p>
+        <form action="https://formspree.io/f/xvodlvop" method="POST">
+            <label for="name">
+							<input type="text" id="name" placeholder="Text Input" size="30" required>
```

Comment:
- [ ] Please consider adding a max-length property on the name input with the max value of 30.